pkg/backends/otlp/internal/data/metric_response_test.go

```diff
@@ -0,0 +1,62 @@
+package data
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1export "go.opentelemetry.io/proto/otlp/collector/metrics/v1"
+	"google.golang.org/protobuf/proto"
+)
+
+func TestProcessMetricsResponse(t *testing.T) {
+	t.Parallel()
+
+	for _, tc := range []struct {
+		name    string
+		body    io.Reader
+		dropped int64
+		errVal  string
+	}{
+		{
+			name:   "Valid response",
+			body:   bytes.NewBuffer(nil),
```

Comment:
Let's make it explicit that we're dropping 0 here even though that is the 0 value. It makes it easier to read at a glance.

Reply:
Easy